src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -103,10 +105,13 @@
       (if transaction-id
         {::accept-transaction {:id           transaction-id
                                :password     password
-                               :on-completed on-transactions-completed}}
-        {:db                (update-in db [:wallet/send-transaction]
-                                       #(assoc % :waiting-signal? true
-                                                 :later? later?))
+                               :on-completed on-transactions-completed}
+         :db (assoc-in db [:wallet/send-transaction :in-progress?] true)}
+        {:db                (-> db
```

Comment:
It could be just
```clojure
(update db :wallet/send-transaction assoc
           :waiting-signal? true
           :later? later?
           :in-progress? true)
```